packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/components/delete/index.js

```diff
@@ -0,0 +1,14 @@
+import angular from 'angular';
+import 'angular-translate';
+import '@ovh-ux/ui-kit';
+
+import component from './component';
+
+const moduleName = 'ovhManagerPoolsComponentsDelete';
+
+angular
+  .module(moduleName, ['oui'])
```

Comment:
Not sure we need the 'oui' module here (I think it is imported from a containing module)

Reply:
Yes